QUEST_GUIDELINES.md

```diff
@@ -5,7 +5,7 @@ Do you have an idea for a new quest? Read this!
 Consider the following:
 
 ### Limitations of StreetComplete
```

Comment:
```suggestion
### Limitations of StreetComplete Quests
```
With that the suggested change is IMO not strictly needed because we have `OVERLAY_GUIDELINES.md`